### PR TITLE
Create new "Hardware" section

### PR DESCRIPTION
**Pull request by jonaharagon**
Changes proposed in this PR:

- Move security keys to separate page
- Add Yubico Security Keys
  - These were already an implied recommendation given that we tell people to use Yubico's quiz to find the right product, but this just splits apart the recommendation into two sections so that we can explain both separately. Plus we can order these security keys first because I think that most people will be fine with FIDO2 functionality only.

partial changes from https://github.com/privacyguides/privacyguides.org/pull/2592

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/basics/passwords-overview.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ There are many good options to choose from, both cloud-based and local. Choose o
 <div class="admonition warning" markdown>
 <p class="admonition-title">Don't place your passwords and TOTP tokens inside the same password manager</p>
 
-When using TOTP codes as [multi-factor authentication](../multi-factor-authentication.md), the best security practice is to keep your TOTP codes in a [separate app](../multi-factor-authentication.md#authenticator-apps).
+When using TOTP codes as [multi-factor authentication](../multi-factor-authentication.md), the best security practice is to keep your TOTP codes in a [separate app](../multi-factor-authentication.md).
```
Comment (redoomed1) [inline docs/basics/passwords-overview.md:167]:
```suggestion
When using [TOTP codes as multi-factor authentication](multi-factor-authentication.md#time-based-one-time-password-totp), the best security practice is to keep your TOTP codes in a [separate app](../multi-factor-authentication.md).
```

File: docs/multi-factor-authentication.md [hunk 2]
```diff
@@ -1,109 +1,22 @@
...
-
-While Nitrokeys do not release the HOTP/TOTP secrets to the device they are plugged into, the HOTP and TOTP storage is **not** encrypted and is vulnerable to physical attacks. If you are looking to store HOTP or TOTP secrets, we highly recommend that you use a YubiKey instead.
-
-</div>
-
-<div class="admonition warning" markdown>
-<p class="admonition-title">Warning</p>
-
-Resetting the OpenPGP interface on a Nitrokey will also make the password database [inaccessible](https://docs.nitrokey.com/pro/linux/factory-reset).
+[Hardware security key recommendations](security-keys.md) have been moved to their own category here.
```
Comment (redoomed1) [inline docs/multi-factor-authentication.md:11]:
```suggestion
[Hardware security key recommendations](security-keys.md) have been moved to their own category.
```

**Review by redoomed1 — CHANGES_REQUESTED** [review]
Aside from the suggestions below, I have another trivial suggestion not related to the content in the PR: modifying the PR title to differentiate this PR from #2268 (or vice versa).

**Comment by redoomed1** [comment]
Also, thoughts on moving the Google Pixel recommendation to this new section (in a different page, of course)? It is currently housed in a section titled ["Android Devices"](https://www.privacyguides.org/en/android/#android-devices), and there is already [separate criteria for Android devices](https://www.privacyguides.org/en/android/#devices). (This change would happen in a separate PR and require that this PR be merged first.)

**Comment by kimg45** [comment]
> Also, thoughts on moving the Google Pixel recommendation to this new section (in a different page, of course)? It is currently housed in a section titled ["Android Devices"](https://www.privacyguides.org/en/android/#android-devices), and there is already [separate criteria for Android devices](https://www.privacyguides.org/en/android/#devices). (This change would happen in a separate PR and require that this PR be merged first.)

Yes that would be awesome

**Comment by jonaharagon** [comment]
I think that this PR will be merged before #2268 so _that one_ will have to be refactored and renamed, but I'll do that after this one is merged.

I think a Mobile Phone page in this new section is a good idea.

**Review by dngray — APPROVED** [review]

**Comment by dngray** [comment]
> I think that this PR will be merged before #2268 so _that one_ will have to be refactored and renamed, but I'll do that after this one is merged.

Certainly, this change makes a lot of sense.

**Review by redoomed1 — APPROVED** [review]
LGTM. Merging this PR also closes #2551.

**Pull request merged**